src/features/school/mentors/mentors.tsx

```diff
@@ -16,7 +16,9 @@ export const Mentors = () => {
             Over the past few years, over 1500+ people have successfully completed our six month
             training program."
           />
-          <Button label="Become a mentor" href="https://app.rs.school/registry/mentor" />
+          <LinkCustom href="https://app.rs.school/registry/mentor" button={true} target="_blank">
```

Comment:
I think this link `https://app.rs.school/registry/mentor` should be encapsulated to app constant

Reply:
Done